packages/nartus_ui_package/lib/widgets/date_label/date_label_view.dart

```diff
@@ -0,0 +1,98 @@
+import 'package:flutter/material.dart';
+
+import 'package:nartus_ui_package/dimens/dimens.dart';
+
+class DateLabelView extends StatelessWidget {
+  const DateLabelView({
+    required this.leadingIcon,
+    required this.dateLabel,
+    required this.tailIcon,
+    Key? key,
+    this.leadingIconColor,
+    this.leadingIconSemanticLabel,
+    this.onLeadingIconPressed,
+    this.labelExpansionIcon,
+    this.dateSemanticLabel,
+    this.tailIconColor,
+    this.tailIconSemanticLabel,
+    this.onTailIconPressed,
+  }) : super(key: key);
+
+  final IconData leadingIcon;
+  final Color? leadingIconColor;
+  final String? leadingIconSemanticLabel;
+  final VoidCallback? onLeadingIconPressed;
+
+  final String dateLabel;
+  final IconData? labelExpansionIcon;
+  final String? dateSemanticLabel;
+
+  final IconData tailIcon;
+  final Color? tailIconColor;
+  final String? tailIconSemanticLabel;
+  final VoidCallback? onTailIconPressed;
+
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+      shape: const RoundedRectangleBorder(
+        borderRadius: BorderRadius.all(Radius.circular(NartusDimens.padding54)),
+      ),
+      color: Theme.of(context).backgroundColor,
+      elevation: NartusDimens.padding4,
+      child: Row(
+        mainAxisSize: MainAxisSize.min,
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: <Widget>[_leadingIcon(), _mainTitle(), _tailIcon()],
```

Comment:
Should we separate each helperMethod here to entirely new Widget according to [this article](https://www.youtube.com/watch?v=IOyq-eTRhvo) from Flutter team ?

Reply:
Good idea, there are onTap action on lead and tail icons, which comes with inkwell effect, so it's better to separate them.

I'll fix this